registration/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.0.5 on 2022-07-18 09:52
+# Generated by Django 4.0.5 on 2022-07-26 08:51
```

Comment:
This alters the initial migration. This should not be done, if there are any working databases with this migration out there.